L1TriggerConfig/Utilities/src/L1TMuonOverlapFwVersionTester.cc

```diff
@@ -0,0 +1,67 @@
+#include <iomanip>
+#include <iostream>
+
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "CondFormats/DataRecord/interface/L1TMuonOverlapFwVersionRcd.h"
+#include "CondFormats/DataRecord/interface/L1TMuonOverlapFwVersionO2ORcd.h"
+#include "CondFormats/L1TObjects/interface/L1TMuonOverlapFwVersion.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+#include "CondCore/CondDB/interface/Session.h"
+
+using namespace std;
+
+class L1TMuonOverlapFwVersionTester : public edm::EDAnalyzer {
+private:
+  bool isO2Opayload;
+  bool writeToDB;
+
+public:
+  void analyze(const edm::Event &, const edm::EventSetup &) override;
+
+  explicit L1TMuonOverlapFwVersionTester(const edm::ParameterSet &pset) : edm::EDAnalyzer() {
+    isO2Opayload = pset.getUntrackedParameter<bool>("isO2Opayload", false);
+    writeToDB = pset.getUntrackedParameter<bool>("writeToDB", false);
+  }
+  ~L1TMuonOverlapFwVersionTester(void) override {}
+};
+
+void L1TMuonOverlapFwVersionTester::analyze(const edm::Event &iEvent, const edm::EventSetup &evSetup) {
+  edm::ESHandle<L1TMuonOverlapFwVersion> handle1;
+  if (isO2Opayload)
+    evSetup.get<L1TMuonOverlapFwVersionO2ORcd>().get(handle1);
+  else
+    evSetup.get<L1TMuonOverlapFwVersionRcd>().get(handle1);
```

Comment:
please follow this
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToGetDataFromES#Getting_data_from_EventSetup_wit

Reply:
Done.